lms/static/js/staff_debug_actions.js

```diff
@@ -53,14 +54,17 @@ var StaffDebug = (function() {
             },
             error: function(request, status, error) {
                 var responseJSON;
+                var errorMsg = _.template(action.error_msg, {interpolate: /\{(.+?)\}/g})(
```

Comment:
What's with the pattern inside interpolate?

Reply:
It has to do with the `{user}` in action.error_msg. Replaces `{user}` with username.